### PR TITLE
Service Registration: Live List Manager

### PR DESCRIPTION
**Pull request by kmh287**
This pull request distinguishes installing/registering the live list manager and creating it.Please see #7839 for more context.

Mostly just function renaming

/to @choumx @jridgewell

### CODE REVIEW

**Comment by kmh287** [comment]
They're not cosmetic. Many services were being instantiated when the caller just needed the service registered. In order to make the distinction, functions called 'install' are just for registering, whereas 'get'FooService' or 'fooServiceForWin' are for actually getting the services.

**Comment by choumx** [comment]
But in this PR, `getLiveListManager` also installs it. AFAIK the only services that call "install" separately are those in `runtime.js`.

**Comment by kmh287** [comment]
It has to to ensure that it's installed the first time the method is called. It's functionally very similar to what was there before. Without this change (and others like it in the coming PRs) some 'install' methods will just register the service and others will return it. I would like to avoid that inconsistent behavior.

File: extensions/amp-live-list/0.1/live-list-manager.js [hunk 1]
```diff
@@ -220,8 +220,16 @@ export class LiveListManager {
...
 export function installLiveListManager(win) {
-  return fromClass(win, 'liveListManager', LiveListManager);
+  registerServiceBuilder(win, 'liveListManager', LiveListManager);
+}
+
+/**
+ * @param {!Window} win
+ * @return {!LiveListManager}
+ */
+export function getLiveListManager(win) {
```
Comment (choumx) [inline extensions/amp-live-list/0.1/live-list-manager.js]: `liveListManagerFor` is more consistent with existing method naming.

**Comment by choumx** [comment]
One practical difference is that we'll be storing the builder closures for many more services than before. We should make sure that they are cleaned up after service instantiation.

**Comment by kmh287** [comment]
That's a good point. I'll need to submit a PR at the end of all of these anyway to clean up the old code paths in service.js. Do you think it can wait until then?

File: extensions/amp-live-list/0.1/live-list-manager.js (hunk 1 above)
Comment (kmh287) [inline extensions/amp-live-list/0.1/live-list-manager.js]: Fair enough. Done.

**Comment by jridgewell** [comment]
> Do you think it can wait until then?

Sure.

**Pull request merged**